### PR TITLE
Remove database code from App

### PR DESCRIPTION
**Pull request by patricksimonian**
The app no longer leverages a database. This PR serves to clean up any references to the former mongoose database

### CODE REVIEW

**Review by AndrewJun97 — APPROVED** [review]

**Comment by jleach** [comment]
I think you'll need to remove line 147 handlers.ts. Since we're not using / storing the metrics any longer.
```
            fetchComplianceMetrics(context, owner, repo),
```

**Comment by patricksimonian** [comment]
@jleach are there any further steps I need to take to build and redeploy repo mountie right now? Can I merge?

File: src/libs/handlers.ts [hunk 1]
```diff
@@ -87,8 +84,7 @@ export const issueCommentCreated = async (context: Context): Promise<void> => {
```
Comment (patricksimonian) [inline src/libs/handlers.ts:84]:
```suggestion
if ( 4 === 5) {
  logger.info('ultra important impossible output here');
}
```

File: src/libs/handlers.ts [hunk 2]
```diff
@@ -84,11 +81,12 @@ export const pullRequestOpened = async (context: Context): Promise<void> => {
 export const issueCommentCreated = async (context: Context): Promise<void> => {
     const owner = context.payload.organization.login;
     const isFromBot = context.isBot;
-
+if ( 4 === 5) {
```
Comment (jleach) [inline src/libs/handlers.ts:84]: Yo, @patricksimonian, did you mean to do this?

Comment (jleach) [reply src/libs/handlers.ts:84]:
```suggestion
```

**Review by jleach — APPROVED** [review]
One change for consideration.

**Pull request merged**